pkgs/desktops/xfce/core/gtk-xfce-engine.nix

```diff
@@ -1,17 +1,19 @@
-{ stdenv, fetchurl, pkgconfig, intltool, gtk }:
+{ stdenv, fetchurl, pkgconfig, intltool, gtk, withGtk3 ? false, gtk3 }:
```

Comment:
Not anymore. Nowadays Xfce is progressively including support for GTK+3 and the next release will be fully migrated to GTK+3. See [Road to Xfce 4.14](http://blog.alteroot.org/articles/2016-04-12/road-to-xfce-4.14.html).

Xfce-4.12 already has some support for GTK+3, including in nixpkgs: [xfce/default.nix](https://github.com/NixOS/nixpkgs/blob/7475728593a49f09c4b7b959b15513aee38ab4b4/pkgs/desktops/xfce/default.nix).

Many applications we run on a desktop these days are based on GTK+3, and they will look ugly with the themes from the Xfce project without the GTK+3 support .